examples/campaign_management/add_campaign_labels.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This code example adds a campaign label to a list of campaigns.
+
+This example assumes that a label has already been prepared.
+"""
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id, label_id, campaign_ids):
+    """This code example adds a campaign label to a list of campaigns.
+
+    Args:
+        client: An initialized GoogleAdsClient instance.
+        customer_id: A client customer ID str.
+        label_id: A label ID to use when searching for campaigns.
+        campaign_ids: A list of campaign IDs to which the label will be added.
+    """
+
+    # Get an instance of CampaignLabelService client.
+    campaign_label_service = client.get_service(
+        'CampaignLabelService', version='v3')
+    campaign_service = client.get_service('CampaignService', version='v3')
+    label_service = client.get_service('LabelService', version='v3')
+
+    # Set the resource name of the label to be added across the campaigns.
+    label_resource_name = label_service.label_path(customer_id, label_id)
+
+    operations = []
+
+    for campaign_id in campaign_ids:
+        campaign_resource_name = campaign_service.campaign_path(customer_id,
+                                                                campaign_id)
+        campaign_label = client.get_type('CampaignLabel', version='v3')
```

Comment:
Is this line really useful? This value is never used (line #55 overrides it with no usage in between).

Reply:
+1 this line is redundant

Reply:
Fixed.